### ISSUE TITLE
Editor Selection Mode status bar widget not working

### ISSUE THREAD

**Issue by nathansolomonspears** [cursive-ide/cursive#2358]
![Screenshot from 2020-04-26 19-21-55](https://user-images.githubusercontent.com/38768862/80325741-9c75f400-87f3-11ea-9204-c699d38b7a59.png)

results in

```
WARN - penapi.actionSystem.impl.Utils - Skipping empty menu item for action  (null) of class com.intellij.openapi.wm.impl.status.widget.StatusBarWidgetsActionGroup$ToggleWidgetAction 
```

in idea.log

All the other widgets seem to work. I removed Cursive and re-installed it, same issue. I just upgraded to IntelliJ 2020.1 and the associated Cursive plugin.

I can still select the editing mode from the Settings menu and it works correctly.

**Comment by cursive-ide** [comment]
This should be fixed in #2349, which has been released in the latest [EAP release](https://cursive-ide.com/userguide/#choosing-to-receive-eap-beta-builds). If that is the build you're using, let me know and I'll re-open this issue.

**Comment by nathansolomonspears** [comment]
Thank you!
